docs/hpc/06_tools_and_software/02_conda_environments.md

```diff
@@ -1 +1,268 @@
 # Conda Environments (Python, R)
+-   [What is Conda?](#what-is-conda)
+-   [Advantages/disadvantages of using Conda](#advantagesdisadvantages-of-using-conda)
+-   [Initializing Conda](#initializing-conda)
+    -   [Automatic deletion of your files](#automatic-deletion-of-your-files)
+-   [Python](#python)
+-   [R](#r)
+-   [Reproducibility](#reproducibility)
+-   [Use conda env in a batch script](#use-conda-env-in-a-batch-script)
```

Comment:
We can skip having links at the top of the page now that we have a second sidebar on the right side of the screen. Google sites only had one sidebar so adding links at the top was likely a work around.

Reply:
Sounds good. Will do.